pages/video-download/video-download.service.js

```diff
@@ -10,10 +10,10 @@ function videoDownloadService(api) {
 
 
 	function get(videoId, code) {
-		api.download(api.url + '/video/' + videoId + '/original?code=' + code, function (data, status, headers) {
-			if(status >= 400 ) {
-				console.error('Unable to download file.');
-			}
-		});
+//		api.download(api.url + '/video/' + videoId + '/original?code=' + code, function (data, status, headers) {
```

Comment:
how do we download the video if we comment the service who does the magic? xDDD 

(I know, it's done to avoid excessive requests while testing, no worry, I'm fixing it now 😆)